app-utils/chafa/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=chafa
+PKGDES="A image-to-text converter for terminal"
+PKGSEC=utils
+PKGDEP="bash libjpeg-turbo librsvg glib cairo libtiff libwebp libjxl freetype libavif "
```

Comment:
There is an unnecessary space at the end

Reply:
Fixed, thanks！